qxf2/newsletter_automation
# Added code to capture flask app metrics

Added integration of Prometheus flask exporter to fetch various metrics from the newsletter application.
The metrics should be visible at `/metrics` endpoint
**To test:**
- Install the requirements
- Run the newsletter application on your local system
- Visit any webpage on the newsletter server
- Go to the metrics page, and search for "flask_http_request_duration_seconds_count"
-  The count for this metric should increment each time you visit the page. For example if you visit `/articles` page then the metric ```flask_http_request_duration_seconds_count{method="GET",path="/articles",status="200"}``` would increment each time you visit the page.
